### PR TITLE
Refactor tile draw functions

### PR DESCRIPTION
**Pull request by juliawu**
The `static/js/chart/draw.ts` has grown very large with the ever increasing number of chart tile types.

This PR refactors that file into:
* 1 `draw_<tile>.ts` file per tile type
* `draw_utils.ts` for shared utility functions
* `draw_constants.ts` for shared constants

Import statements are updated to reflect the new filenames.

We should also do a followup cleanup of docstrings for each exported function. We can add that in a followup PR, after the current features push.

Testing: verified search interface, visualization tools, and web components didn't have any change in rendering behavior.

### CODE REVIEW

**Review by chejennifer — COMMENTED** [review]
thanks for refactoring!

File: static/js/chart/draw_utils.ts [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+        rowLabel += _.isEmpty(rowLabel) ? `${place}` : ` (${place})`;
+      }
+      labels[place][dataGroup] = rowLabel;
+    }
+  }
+  return labels;
+}
+
+/**
+ * Gets the html content of a tooltip
```
[the remark this answers is not in the document]
Comment (juliawu) [reply static/js/chart/draw_utils.ts]: This sounds good to me. Moved to `draw_line.ts`.

File: static/js/chart/draw_constants.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Shared constants used in the draw functions for chart tiles
+ */
+
+export const AXIS_GRID_FILL = "#999";
```
[the remark this answers is not in the document]
Comment (juliawu) [reply static/js/chart/draw_constants.ts]: Sounds good. Moved constants specific to draw_utils into `draw_utils.ts`.

**Review by chejennifer — APPROVED** [review]
thanks for updating!

**Pull request merged**